python/src/DistributionImplementation_doc.i.in

```diff
@@ -160,15 +159,17 @@ With respect to the other fixed components.
 
 Parameters
 ----------
-Xn : float
-    Conditional DDF input (last component).
-Xcond : sequence of float with dimension :math:`n-1`
-    Conditioning values for the other components.
+Xd : float
+    Real in :math:`\Rset`.
```

Comment:
A real number is necessarily in :math:`\Rset`: this does not bring much information.
```suggestion
    The input value where the method must be evaluated.
```